text/0032-access_command.md

```diff
@@ -173,26 +182,29 @@ pushing 28 versions of chriswhong/some_dataset...
 X of XX blocks pushed for version 12 August 2019 (Qmss4h552h3j2h33)...
 ```
 
-A `--hidden` flag on `push` overrides this behaviour, which will present a warning:
+Defaulting to `visible` avoids a potential UX problem when a dataset is pushed: without setting a dataset to visible everyone (including the user) will not be able to see the pushed dataset in list operations on the remote after pushing. Users who want to push hidden datasets should first make them hidden with `$ qri access hidden`, then push. Pushing a hidden dataset should present a warning:
```

Comment:
I think the long term solution is driving a "who is asking" parameter deeper into remote behavior. We'll _need_ this as a part of access control, but it could also remove the need for the "you're pushing a hidden dataset" warning, because remotes can show users lists contextualized to their identities.

Because the registry is a remote, it would inherit this behavior